zappa/core.py

```diff
@@ -1209,6 +1221,7 @@ def update_lambda_function(
         num_revisions=None,
         concurrency=None,
         docker_image_uri=None,
+        architecture=None,
```

Comment:
Since we have already initialized Zappa with the architecture attribute, can we remove the architecture parameter here?